nasa/harmony
# Harmony 1130 unpause

## Jira Issue ID
HARMONY-1130

## Description
Adds a resume endpoint for normal users and admin users. 

## Local Test Steps
* Start a long-running (async) job
* Change the state of the job to 'paused' in the database
* Connect to the job status page and verify that the job is paused
* Resume the job using the `/jobs/:jobID/resume` endpoint
* Connect to the job status page and verify that the job is running

## PR Acceptance Checklist
* [ ] ~Documentation updated (if needed)~ - will need to wait until auto-pause is added as there is currently no way for a user to pause their own job, therefore no way for a user to make use of the resume endpoint right now
